content/develop/data-types/probabilistic/configuration.md

```diff
@@ -0,0 +1,166 @@
+---
+categories:
+- docs
+- develop
+- stack
+- oss
+- rs
+- rc
+- oss
+- kubernetes
+- clients
+description: Redis probabilistic data structures support multiple configuration parameters.
+linkTitle: Configuration
+title: Configuration Parameters
+weight: 100
+---
+{{< note >}}
+As of Redis Community Edition 8.0, configuration parameters for the probabilistic data structures are now set in the following ways:
+* At load time via your `redis.conf` file.
+* At run time (where applicable) using the [`CONFIG SET`]({{< relref "/commands/config-set" >}}) command.
+{{< /note >}}
```

Comment:
We can mention that starting with v8.0 the probabilistic configuration parameters will be persisted in the same way as any other Redis configuration parameters (with `CONFIG REWRITE` for CE).

Reply:
Applied in next push.